triples-generator/src/main/scala/io/renku/triplesgenerator/Microservice.scala

```diff
@@ -68,10 +69,11 @@ object Microservice extends IOMicroservice {
         gitLabRateLimit                <- RateLimit.fromConfig[IO, GitLab]("services.gitlab.rate-limit")
         gitLabThrottler                <- Throttler[IO, GitLab](gitLabRateLimit)
         sparqlTimeRecorder             <- SparqlQueryTimeRecorder[IO]
+        gitLabClient                   <- GitLabClient[IO](gitLabThrottler)
         awaitingGenerationSubscription <- events.categories.awaitinggeneration.SubscriptionFactory[IO]
         membersSyncSubscription <- events.categories.membersync.SubscriptionFactory(gitLabThrottler, sparqlTimeRecorder)
         triplesGeneratedSubscription <-
-          events.categories.triplesgenerated.SubscriptionFactory(gitLabThrottler, sparqlTimeRecorder)
+          events.categories.triplesgenerated.SubscriptionFactory(gitLabThrottler, sparqlTimeRecorder, gitLabClient)
```

Comment:
Actually, do we need to pass the Throttler to the factories once we started using GitLabClient?

Reply:
The GitLabProjectMembersFinderImpl still uses it at the moment. I can refactor that class so we can completely remove it.